optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-8.md

```diff
@@ -13,6 +13,22 @@ description: "Connection to Camunda 8."
 | zeebe.partitionCount    | 1             | The number of partitions configured for the Zeebe record source.                                                             |
 | zeebe.maxImportPageSize | 200           | The max page size for importing Zeebe data.                                                                                  |
 
+### Licensing
+
+<span class="badge badge--platform">Camunda 8 Self-Managed only</span>
+
+Installations of Camunda 8 which require a license can provide their license key to Optimize as an environment variable:
+
+| Environment variable  | Description                                                          | Default value |
+| --------------------- | -------------------------------------------------------------------- | ------------- |
+| `CAMUNDA_LICENSE_KEY` | Your Camunda 8 license key, if your installation requires a license. | None          |
+
+For Helm installations, license keys can be configured globally in your `values.yaml` file. See the [Helm installation documentation]($docs$/self-managed/setup/install#configure-license-key) for more details.
+
+:::note
+Camunda 8 components without a valid license may display **Non-Production License** in the navigation bar and issue warnings in the logs. These warnings have no impact on startup or functionality.
+:::
```

Comment:
On this page: https://docs.camunda.io/docs/next/self-managed/setup/deploy/other/docker/#optimize

There is a property called `CAMUNDA_OPTIMIZE_ENTERPRISE`. I believe that this would now be redundant in C8. Shall we remove it there? I cannot find equivalent pages for the other components

Reply:
All set here: https://github.com/camunda/camunda-docs/issues/4131